db/data/seedDatabase.js

```diff
@@ -12,22 +12,22 @@ MongoClient.connect(url, (err, client) => {
 
     let id = 0;
     let fakeRestaurantsBatch;
-    const batchSize = 10000;
+    const batchSizeSetting = 10000;
```

Comment:
for values like batchSizeSettings and numBatches, it may be worth grabbing from an environment variable but still setting a default, such as `const batchSizeSetting = parseInt(process.env.BATCH_SIZE, 10) || 10000;` That way you still have a backup default number, but you can dynamically change it at the time you run the script by typing something like this into bash: `BATCH_SIZE=5000 NUM_BATCHES=1000 node db/data/seedDatabase.js`

Again more of a nice-to-have as it doesn't critically affect functionality; but I noticed that you changed the numBatches below, and using environment variables allows you to avoid changing hardcoded values when you want to run a script with different values! Maybe worth taking a look at in other areas such as the server if you want to move on from editing the seed script 😄

Reply:
Great idea, I should get more comfortable with environment variables. I've made this change.